src/main/resources/sql/migrations/tenant/V46__Add_audit_columns_in_variables.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE variables
```

Comment:
@shejasabin does this mean the script here was not executed https://github.com/one-acre-fund/oaf--ph-ee-operations-app/pull/19/files